client/egypt-drug-checker-dashboard/src/components/home/content-container/ContentContainer.component.tsx

```diff
@@ -0,0 +1,49 @@
+import { type IcontentContainer } from "./ContentContainer.model";
+import { Typography } from "@mui/material";
```

Comment:
same as before

Reply:
chore(frontend): adjusting name of the typography component to MUtypography